test/unit/application.spec.js

```diff
@@ -65,20 +65,6 @@ describe('marionette application', function() {
     });
   });
 
-  describe('when an app has been started, and registering another initializer', function() {
```

Comment:
> a clarifying question, maybe: what's the plan to support loading sub-apps at runtime, and having them start up?

It's a hope of mine that we stop promoting 'sub-apps' to users of Marionette in the near future.

This is beyond the scope of this PR, but it's my belief that with a stateful router and services pattern, we would find that there's no need for 'container' objects like "Sub Applications." The Application itself should just be an object you create once to get setup with some of the most common features of a Marionette app, like a router.